torch_dipu/csrc_dipu/binding/DIPUpybind.h

```diff
@@ -0,0 +1,65 @@
+#include <pybind11/pybind11.h>
+#include <torch/types.h>  // for at::ScalarType
+#include <torch/python.h> 
+
+namespace py = pybind11;
+
+namespace pybind11 {
+namespace detail {
+
```

Comment:
global namespace alias in header is not good.

```suggestion
namespace pybind11 {
namespace detail {

namespace py = ::pybind11;

```

Reply:
have fixed it